apps/web/public/static/locales/nl/common.json

```diff
@@ -172,9 +172,9 @@
   "link_expires": "p.s. De uitnodiging verloopt over {{expiresIn}} uur.",
   "upgrade_to_per_seat": "Upgraden naar per plaats",
   "seat_options_doesnt_support_confirmation": "De optie plaatsen optie ondersteunt geen bevestigingseis",
-  "multilocation_doesnt_support_seats": "Meerdere Locaties ondersteunt geen zitplaatsenoptie",
-  "no_show_fee_doesnt_support_seats": "No-show vergoeding ondersteunt geen zitplaatsenoptie",
-  "seats_option_doesnt_support_multi_location": "Zitplaatsenoptie ondersteunt geen Meerdere Locaties",
+  "multilocation_doesnt_support_seats": "Meerdere Locaties ondersteunt geen plaatsenoptie",
```

Comment:
But isn't 'plaatsenoptie' a physical place or location, and everywhere I've seen, 'zitplaatsen' specifically defines seats?

Reply:
not really. Plaatsen can refer to a physical location as well, but also a spot in a queue for example. zitplaatsen refers specifically to seated spots, so for example if you want to reserve a spot at your local sportsclub, then 'plaats' would be a better translation. Unless the purpose of the seats is only for physical seats but I don't see why you would limit yourself to just that.

Reply:
cc @emrysal

Reply:
@SDeVuyst is correct - zitplaats has a specific meaning such as a "physical seat" at a concert, or at a table, it may have "12 zitplaatsen" - strictly speaking it's not wrong - however, "plaatsen" should still be correctly interpreted by the reader as how many places are still available within the context of the event (you're meant to "place yourself" for an event, not to be seated for an event). Long story short, it's correct.